.github/workflows/tests.yml

```diff
@@ -34,15 +34,15 @@ jobs:
           cache: false # don't use cache for self-hosted runners
 
       - name: Unit Test
-        run: go test -race -covermode=atomic -coverprofile=coverage.txt ./...
+        run: go test -race -ldflags=-checklinkname=0 -covermode=atomic -coverprofile=coverage.txt ./...
```

Comment:
flag provided but not defined: -checklinkname

Reply:
yeah, emmm, this is 1.23 only

Reply:
still better to wait for `github.com/bytedance/mockey`'s new version.